utils/oc.py

```diff
@@ -230,7 +230,10 @@ def get_api_resources(self):
         # oc api-resources only has name or wide output
         # and we need to get the KIND, which is the last column
         cmd = ['api-resources', '--no-headers']
-        results = self._run(cmd).decode('utf-8').split('\n')
+        if self._run(cmd) == '{}':
+            results = '{}'
+        else:
+            results = self._run(cmd).decode('utf-8').split('\n')
```

Comment:
This is basically saying that the cluster has no resources in it, but it will still allow the openshift-resources to apply whaterver is specified in app-interface, which it shouldn't.
Also, this mechanism depends on `init_api_resources` to be True on OC init.
Last but not least, this is breaking things:
```
...
  File "/home/apahim/git/qontract-reconcile/reconcile/openshift_base.py", line 231, in apply
    oc.recycle_pods(dry_run, namespace, resource_type, resource)
  File "/home/apahim/git/qontract-reconcile/utils/oc.py", line 291, in recycle_pods
    pods = self.get(namespace, 'Pods')['items']
KeyError: 'items'
```
Did you test this properly?

Reply:
When I ran this I got zero errors back.  I assigned a cluster DNS FQDN to point to my laptop.  In both positive and negative testing I never got these errors, but maybe my setup is incorrect.